include/variables.h

```diff
@@ -25,7 +25,7 @@ extern s8    gHumanPlayerCount;
 extern f32   gKartFrictionTable[];
 extern f32   gKartBoundingBoxTable[];
 extern f32   gKartGravityTable[];
-extern s8    gMenuState;                            // D_8018EDF0
+extern s8    gControllerPakMenuSelection;           // D_8018EDF0
```

Comment:
I see why I thought this was related to the main menu now. I forgot this was an s8, not an s32, which led to me misinterpreting the debugger output. The name given here is much better

Reply:
Ahhh ok. Wasn't sure if I was missing something. Glad this isn't some weird dual-use variable